README.md

```diff
@@ -20,7 +20,7 @@ Other MkDocs plugins that use information from git:
 
 ## Setup
 
-Install the plugin using `pip3` with the following command:
+Install the plugin using `pip3` with the following command (if running in Docker then you will need to add this to the `Dockerfile` that builds the image):
```

Comment:
Not entirely true, as you have to use the prefix `CMD` also. 

On second thought, the comment 'built in support for' is misleading. I went ahead and rewrote the section (https://github.com/timvink/mkdocs-git-revision-date-localized-plugin/commit/71ced230cde08981ca98267106002c98f7dd72c5). 

Hope you don't mind me closing this PR

Reply:
Ah yes good point re `CMD`